propolis/src/hw/nvme/mod.rs

```diff
@@ -466,7 +471,7 @@ impl PciNvme {
         // Initialize the Identify structure returned when the  host issues
         // an Identify Namespace command.
         let total_bytes = binfo.total_size * binfo.block_size as u64;
-        let nsze = total_bytes / BLOCK_SZ;
+        let nsze = total_bytes / (binfo.block_size as u64);
```

Comment:
We can probably simplify this and just do:
```rust
let nsze = binfo.total_size;
```

Reply:
Yeah... good catch. simplified in 14f4dc9.